modules/virt-storage-wizard-fields-web.adoc

```diff
@@ -72,6 +80,10 @@ endif::[]
 |Name | Parameter |  Description
 
 .2+|Volume Mode
+[NOTE]
+====
+Default values are used from the storage profile.
```

Comment:
Suggest tweaking the colwidths (possibly in both tables, but definitely this one) because they're getting pretty squished w/ the NOTE boxes:

![image](https://user-images.githubusercontent.com/3442316/135172638-03e44c07-777c-4700-bfa6-a03b9573fba9.png)

It looks like there's a lot of extra room in the 2nd column that could probably be given to these first columns here.

Reply:
Changing both table col defs to 5a-2a-5a. Can you verify this does not override any individual cell, row, column defaults? And let me know if it just looks bad to you and we can try again.